app/Http/routes.php

```diff
@@ -12,6 +12,7 @@
     Route::get('conferences/{conference}', 'ConferencesController@show');
 });
 
+Route::post('api/conferences/{conference}/introduction', 'ConferenceIntroductionController@store');
```

Comment:
Hm, I'm guessing you did this so you didn't have to have two controllers with a single method? I'm torn about whether I like it this way or not.